src/main/resources/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel/help-durabilityHint.html

```diff
@@ -0,0 +1,20 @@
+<div>
```

Comment:
Please use a Title-cased "Pipeline" or "Pipelines" in this document to maintain consistency with other aspects of Jenkins Pipeline in the application.